R/03-plotModule.R

```diff
@@ -0,0 +1,42 @@
+#' UI function of plot module
+#'
+#' @param id id of the module
+plotUI <- function(id) {
+  ns <- NS(id)
+  plotOutput(ns("plot"))
+}
+
+
+#' Server function of plot module
+#'
+#' @param id id of the module
+#' @param path path to the file
+#' @param file_type file type of file
+#' @param variable variable from user selection
+#' @param measure measure from user selection
+#' @param time time from user selection
+plotServer <- function(id, path, file_type, variable=NULL, measure=NULL, time=NULL) {
+  moduleServer(
+    id,
+    function(input, output, session) {
+      if(file_type == "png"){
```

Comment:
Is it possible that `file_type` has value `NULL`? E.g. if loading fails or so? Than the if-clause would crash...

Reply:
Now the file type is checked as well when the json is loaded.